src/main/java/uk/nhs/prm/repo/ehrtransferservice/config/SnsClientSpringConfiguration.java

```diff
@@ -2,25 +2,32 @@
 
 import com.amazonaws.services.sns.AmazonSNS;
 import com.amazonaws.services.sns.AmazonSNSClientBuilder;
+import lombok.RequiredArgsConstructor;
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
+import software.amazon.awssdk.auth.credentials.AwsCredentialsProvider;
 import software.amazon.awssdk.regions.Region;
 import software.amazon.awssdk.services.sns.SnsClient;
 
 @Configuration
-
+@RequiredArgsConstructor
 public class SnsClientSpringConfiguration {
     @Value("${aws.region}")
     private String awsRegion;
 
+    private AwsCredentialsProvider awsCredentialsProvider;
+
     @Bean
     public AmazonSNS snsClient() {
         return AmazonSNSClientBuilder.standard().withRegion(awsRegion).build();
```

Comment:
Could we break this into other lines if possible to keep it consistent?

```suggestion
        return AmazonSNSClientBuilder.standard()
        .withRegion(awsRegion)
        .build();
```